tasks/trace_agent.py

```diff
@@ -83,3 +83,28 @@ def integration_tests(ctx, install_deps=False, race=False, remote_docker=False):
 
     for prefix in prefixes:
         ctx.run("{} {}".format(go_cmd, prefix))
+
+@task
+def cross_compile(ctx, tag=""):
+    """
+    Cross-compiles the trace-agent binaries. Use the "--tag=X" argument to specify build tag.
+    """
+    if tag == "":
+        print("Argument --tag=<version> is required.")
+        return
+
+    print("Buliding tag %s..." % tag)
```

Comment:
Just a nit but maybe stick to one formatting method?

Reply:
I prefer to have this on because it's shorter. In the other group I'd like to stay consistent throughout and use `.format(` because some lines have `{tag}` occurring 2-4 times and it doesn't look too good (imho) to have `% (tag,tag,tag,tag)`

Reply:
Haha, I understand - `% (tag, tag, tag, tag)` is a little tough on the eyes.